### ISSUE TITLE
Image Rotation

### ISSUE THREAD

**Issue by des-coder** [guigrpa/docx-templates#221]
Can you add an optional image type for rotation.

/src/types.ts 

insert into ImagePars Line 222:
```
  /**
   * Optional rotation in degrees.
   */
  rotation?: number;
```

/src/processTemplate.ts

insert into processImage Line 801:

`const rot = (imagePars.rotation * 60e3).toString() || "0";`

Edit line 853

`node('a:xfrm', {}, [` 
-->
`node('a:xfrm', {rot: rot}, [`

**Comment by jjhbw** [comment]
Good idea! Seems like a nice simple addition. I've added this to master with 64cd5784623e3d55c58e914b29897550d6eaff26.

It will be a part of the next release. Thanks for the suggestion.

**Comment by jjhbw** [comment]
Fix released in v4.7.0. Should be on NPM shortly.
